testsuite/oidc/rhsso/__init__.py

```diff
@@ -0,0 +1,76 @@
+"""Objects for RHSSO"""
+from functools import cached_property
+from urllib.parse import urlparse
+
+from keycloak import KeycloakOpenID, KeycloakAdmin
+
+from testsuite.oidc import OIDCProvider, Token
+from testsuite.objects import LifecycleObject
+from .objects import Realm, Client
+
+
+# pylint: disable=too-many-instance-attributes
+class RHSSOServiceConfiguration(OIDCProvider, LifecycleObject):
```

Comment:
Maybe keep the naming with `Auth0Provider`? 
Also reformat doc string to be one-liner